repos/core-setup.proj

```diff
@@ -26,28 +25,14 @@
 
   <UsingTask AssemblyFile="$(TasksBinDir)Microsoft.DotNet.SourceBuild.Tasks.dll" TaskName="PublishCoreSetupBinaries" />
 
-  <!-- We don't build the UAP Package so we need to remove it from the list of
-       packages we try to update from, since it will not be in our Packages.txt
-       file -->
-  <Target Name="RemoveUAPFromUpdateSteps" BeforeTargets="UpdateDependencies">
-    <ItemGroup>
-      <FilteredUpdateStep Include="@(UpdateStep)" Condition="'%(PackageId)' != 'Microsoft.Private.CoreFx.UAP'" />
-      <UpdateStep Remove="@(UpdateStep)" />
-      <UpdateStep Include="@(FilteredUpdateStep)" />
-    </ItemGroup>
-  </Target>
-
+  <!-- Need to "publish" the tarballs for cli, until cli respects the assets blob -->
   <Target Name="PublishBinaries" AfterTargets="Package">
     <ItemGroup>
-      <CoreSetupTarballs Include="$(BinariesOutput)*$(TarBallExtension)" />
+      <CoreSetupTarballs Include="$(SourceBuiltAssetsDir)dotnet-runtime-*$(TarBallExtension)" />
```

Comment:
I didn't realize it Core-Setup-side--this path means https://github.com/dotnet/source-build/blob/2c79047d76f6a1feb4cc70077102d3409cea9634/Documentation/auto-dependency-flow/api.md#-pdotnetassetrooturlurl-not-yet-implemented should be updated. Currently the recommended implementation is specifically for CLI and it says the installers will be in a `Runtime/{version}/` dir within the blob feed.

(It isn't important to me which way we go--tried both in the PRs for that doc and ended up where we are now, but all that matters is being consistent.)

Reply:
This also impacts how we are going to do publishing in orchestrated builds.  I'm not sure I know which way is the "right" way here.

Reply:
One thought is, if we have a single "assets root url" for a build, it doesn't really make sense to put things in a subfolder under the root it with a version in the folder.  The root url would already be unique for the given build, right?

(I know I may have said the opposite previously, but I don't think I was realizing all the intricacies here.)

Reply:
The root url is unique for each product build, but we *could* find ourselves building multiple versions of Core-Setup within that product build at some point. As long as Core-Setup maps each file directly to a target publish location, we're fine, but I don't know if there's any globbing we expect to be able to do.

My gut agrees that we shouldn't need this "extra" `Runtime/{version}/` dir because the version is already in the filename at this point. (Only removed once we publish to latest on `dotnetcli` storage.)

Reply:
I updated the doc to the current approach.  We can make adjustments as we go, if we decide this is a bad approach.